### DISCUSSION TITLE
Shiki not load on production

### DISCUSSION THREAD

**Discussion by hilmanski** [spatie/laravel-markdown#60] [Q&A] [answered]
Hi
I use laravel-markdown, it works well. 

On local highlich code shikin work perfectly.
But once is's on produciton, nothing happen to the code, no error either

**Comment by hilmanski** [comment]
I guess right now the problem is
it render my code as <code> $code </code> in produciton

while in local it has `<pre> tag` and other shiki stuff 
I'm not sure why because the exact same data

I even tried run `npm install shiki` directly in production
and nothing happen

**Reply by d76g** [reply]
@hilmanski , did you solve this ?

**Comment by WendellAdriel** [comment]
The same is happening with me

**Comment by WendellAdriel** [comment, marked as answer]
I had to create a link from `/usr/bin/node` to `/usr/local/bin/node` and it worked!

**Comment by magarrent** [comment]
After two days of debugging, I got it!

Apart from @WendellAdriel response, I've found that I had this error:

` is_executable(): open_basedir restriction in effect. File(/usr/bin/node) is not within the allowed path(s)`

So if it is your case, just add the path to the PHP allowed path!

**Comment by d76g** [comment]
I am hosting my Laravel Project in Hostinger, and I am facing the same problem. Is there in solution for this for Shared Hosting?

**Reply by magarrent** [reply]
I don't think so. I don't recommend shared hosting for Laravel projects never, you might have a lot of troubles. But shiki uses nodejs and I don't think a Shared Hosting supports that
